src/next/blocks/blockReleaseIt.ts

```diff
@@ -31,6 +31,10 @@ export const blockReleaseIt = base.createBlock({
 									types: ["published"],
 								},
 							},
+							permissions: {
+								issues: "write",
+								"pull-requests": "write",
+							},
```

Comment:
😁 I do think it kind of painfully funny that in attempting to remove one annoying end-to-end test (`script/migrate-test-e2e.test.ts`) I'm temporarily giving you another probably annoying extra bit of work... thanks for tolerating it and also contributing to the new block engine!

Reply:
"painfully funny" is accurate!

I've come to regard the automated tests of CTA as developer hostile end-of-level baddies. If I was into darts I'd print out one of the failed test runs and pin it to the board 😅